app/common/static/css/chat_popup.css

```
@@ -210,35 +210,45 @@
 
+/* Message Bubbles */
 /* Message Bubbles */
 .chat-message {
     max-width: 85%;
+    width: fit-content;
     padding: 12px 18px;
     border-radius: 18px;
     font-size: 0.95rem;
     line-height: 1.5;
     position: relative;
     word-wrap: break-word;
```

Stylelint (17.3.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)